contributing-docs/14_node_environment_setup.rst

```diff
@@ -18,6 +18,64 @@
 Node.js Environment Setup
 =========================
 
+Contributing to the UI in Airflow
```

Comment:
I wanted to try and also found the docs in `airflow/ui/CONTRIBUTING.md` - does it make sense to cross-link here and remove the redundancy?